person.rb

```diff
@@ -27,4 +28,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
```

Comment:
- [ ] I would highly recommend refining this method for adding rentals by having it accept both `date and book` parameters. 📆👤 Then, when you call the Rental.new constructor, you can pass self as the `person` parameter. This approach will effectively establish a "has-many" association between your `person` and `rentals`. 📚👥

```suggestion
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```